crates/stackable-operator/CHANGELOG.md

```diff
@@ -34,7 +34,12 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - Update `kube` to `1.1.0` ([#1049]).
-- BREAKING: Return type for `ListenerOperatorVolumeSourceBuilder::new()` is no onger a `Result` ([#1058]).
+- BREAKING: Return type for `ListenerOperatorVolumeSourceBuilder::new()` is no longer a `Result` ([#1058]).
+- BREAKING: Require two new CLI arguments: `--operator-namespace` and `-operator-service-name`.
+  These are required, so that the operator knows what Service it needs to enter as CRD conversion webhook ([#1066]).
```

Comment:
note: This is an addition and as such needs to be moved to the "Added" section. Also, slight reword:


```suggestion
- BREAKING: Add two new required CLI arguments: `--operator-namespace` and `-operator-service-name`.
  These two values are used to construct the service name in the CRD conversion webhook ([#1066]).
```